src/quo2/components/tabs/tabs/view.cljs

```diff
@@ -166,7 +167,8 @@
              ;; from ScrollView (e.g. FlatList). There are open issues, here's
              ;; just one about this topic:
              ;; https://github.com/facebook/react-native/issues/31218
-             :content-container-style           {:padding-top (dec unread-count-offset)}
+             :content-container-style           (merge content-style
```

Comment:
`assoc` is a good choice here.

Reply:
should we call this prop `container-style` as it's essentially the outer later? 🤔

Reply:
`content-container-style` is most accurate actually, but we can simplify with `container-style` yes

Reply:
Well whatever is most accurate, container-style is what we're referring to add to the most outer view to add margin etc,